### PR TITLE
Feature/aws

### PR DESCRIPTION
**Pull request by AamirL1011**
Modifications to accommodate AWS S3 image uploading.

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -3,7 +3,18 @@ const router = express.Router();
 const ObjectID = require("mongodb").ObjectID;
 const multer  = require('multer');
 const upload = multer({ dest: 'uploads/' });
-const { uuidv4 } = require('uuidv4');
+const { uuid } = require('uuidv4');
```
Comment (rhiknow55) [inline routes/users.js:6]: thx for fixing lol

Comment (rhiknow55) [reply routes/users.js:6]: have you tested it with registration?

**Review by rhiknow55 — APPROVED** [review]
coooooool

File: routes/users.js (hunk 1 above)
Comment (AamirL1011) [reply routes/users.js:6]: I tested it with a new registration, it seems to work.

**Pull request merged**